README.md

```diff
@@ -16,7 +16,7 @@ Role Variables
 --------------
 
 - omero_ms_image_region_user: 'omero-server': icroservice user, the defaults is omero-server
```

Comment:
```suggestion
- omero_ms_image_region_user: Microservice user, the default is `omero-server`
```

Maybe like this ^^^ ? But double-check please, this particular definition seems to be duplicate one line below and the intention is not clear.